vendor/views/config.py

```diff
@@ -0,0 +1,280 @@
+        context['form'].fields['site'].disabled = True
+        context['cancel_url'] = reverse('vendor_admin:manager-config-stripe-connect-list')
+
+        return context
+
+
+    def post(self, request, *args, **kwargs):
+        # Get the SiteConfigModel instance passed through the url
+        self.object = self.get_object()
+        context = super().get_context_data(**kwargs)
+
+        form = self.get_form_class()(request.POST)
+        # Make this site field required False since you can only update the account_number
+        form.fields['site'].required = False
+
+        if not form.is_valid():
+            context['form'] = form
+            return render(request, self.template_name, context)
+        
+        stripe_connect = StripeConnectAccountConfig(self.object.site)
+        stripe_connect.save(form.cleaned_data['account_number'], "stripe_connect_account")
+
+        return redirect("vendor_admin:manager-config-stripe-connect-list")
+
+
+class VendorSiteCommissionCreateConfigView(FormView):
+    template_name = 'vendor/manage/config.html'
+    form_class = VendorSiteCommissionForm
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        commission_config = VendorSiteCommissionConfig()
+        form = self.get_form()
+
+        existing_commission_configs = SiteConfigModel.objects.filter(key=commission_config.key)
+        sites = Site.objects.exclude(pk__in=[config.site.pk for config in existing_commission_configs])
+        form.fields['site'].queryset = sites
+
+        context['title'] = _("Vendor Site Commission")
+        context['form'] = form
+        context['cancel_url'] = reverse('vendor_admin:manager-config-commission-list')
+
+        return context
+
+    def form_valid(self, form):
+        site = form.cleaned_data['site']
+        
+        commission = VendorSiteCommissionConfig(site)
+        commission.save(form.cleaned_data['commission'], "commission")
+
+        return redirect("vendor_admin:manager-config-commission-list")
+
+
+class VendorSiteCommissionConfigListView(ListView):
+    template_name = 'vendor/manage/config_list.html'
+    model = SiteConfigModel
+
+    def get_queryset(self):
+        commission_config = VendorSiteCommissionConfig()
+        
+        return self.model.objects.filter(key=commission_config.key)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        commission_config = VendorSiteCommissionConfig()
+
+        context['title'] = 'Vendor Commissions'
+        context['config_key'] = commission_config.key
+        context['new_url'] = reverse('vendor_admin:manager-config-commission-create')
+
+        return context
+
+
+class VendorSiteCommissionUpdateConfigView(UpdateView):
+    template_name = 'vendor/manage/config.html'
+    model = SiteConfigModel
+    form_class = VendorSiteCommissionForm
+
+    def get_form(self, **kwargs):
```

Comment:
same note as before

Reply:
removed kwargs

Reply:
I see what you mean duh. thanks

Reply:
I remember now. I am over righting it because I want it to ignore built in functionality it has whit the UpdateView Class